include/openPMD/ParticleSpecies.hpp

```diff
@@ -45,6 +45,16 @@ class ParticleSpecies : public Container< Record >
 
     void read();
     void flush(std::string const &) override;
+
+    /**
+     * @brief Verify that a particle species in a closed iteration has not
+     *        been wrongly accessed.
+     *
+     * @return true If closed iteration had no wrong accesses.
+     * @return false Otherwise.
+     */
+    bool
+    verifyClosed() const;
```

Comment:
see comment above about naming and behavior when called on a still-open `ParticleSpecies`